course_catalog/tasks_test.py

```diff
@@ -290,3 +297,69 @@ def test_get_youtube_data(mocker, settings, channel_ids):
     mock_pipelines = mocker.patch("course_catalog.tasks.pipelines")
     get_youtube_data.delay(channel_ids=channel_ids)
     mock_pipelines.youtube_etl.assert_called_once_with(channel_ids=channel_ids)
+
+
+@pytest.mark.parametrize("overwrite", [True, False])
+@pytest.mark.parametrize("created_after", ["2019-10-01T00:00:00Z", None])
+@pytest.mark.parametrize("created_minutes", ["200", None])
+def test_get_youtube_videos_for_transcripts_job(
+    mocker, created_after, created_minutes, overwrite
+):
+    "Verify that get_youtube_videos_for_transcripts_job applies filters correctly"
+    mock_videos = mocker.patch("course_catalog.models.Video.objects")
```

Comment:
You don't want to mock the ORM here since it avoids testing that the calls being made are valid ORM operations, instead you should create the records in the test db and then verify that the resulting queryset contains the objects you expect based on the parameters you're passing in.